backend/x11/output.c

```diff
@@ -357,8 +357,7 @@ struct wlr_output *wlr_x11_output_create(struct wlr_backend *backend) {
 	struct wlr_output *wlr_output = &output->wlr_output;
 	wlr_output_init(wlr_output, &x11->backend, &output_impl, x11->wl_display);
 
-	wlr_output->width = 1024;
-	wlr_output->height = 768;
+	wlr_output_update_custom_mode(wlr_output, 1024, 768, 60);
```

Comment:
The refresh rate can be set to 0 to mean "undefined". We use Present to drive the rendering loop, so it's not fixed.

Reply:
OK. Changed to 0.